### DISCUSSION TITLE
How disable operator "AND"/"OR" in multi-filtering?

### DISCUSSION THREAD

**Discussion by subobuild** [mui/mui-x#2864] [Archive]
My tables have server-side filtering. I need to disable operators from multi-filtering popup menu (check the screenshot). It Is possible? It is possible also to remove it? If yes, how?  Thanks in advance
![operators](https://user-images.githubusercontent.com/10583162/137270161-55b72c94-b4a5-43a7-ab42-73d18a311ea3.png)

**Comment by m4theushw** [comment]
Yes, you can show only the operators you want, see https://mui.com/components/data-grid/filtering/#extend-filter-operators. To remove the operator selector, you can use CSS and control the `filterModel` prop to not allow to have more than one filter active.

**Reply by subobuild** [reply]
Thanks
